PovertyInfo/src/interactiveinfo/Program.java

```diff
@@ -0,0 +1,35 @@
+package interactiveinfo;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import org.apache.commons.lang3.text.WordUtils;
+
+public class Program {
+
+	public static void main(String[] args) throws FileNotFoundException {
+		Scanner scnr = new Scanner(System.in);
+		System.out.println("Enter a county name in Washington (e.g., King), or type `quit` to quit: ");
+		String input = WordUtils.capitalizeFully(scnr.nextLine().trim());
+				
+		String record = null;
+		while (!input.equals("Quit") && !input.equals("Q")) {
+			record = GetPovertyInfo.getMatch(input);
+			
+			while (record == null) {
+				scnr = new Scanner(System.in);
```

Comment:
You could adjust to have record start out as null and put all of the code from lines 11-13 in this while look as well so that none of it is duplicated.